package.json

```diff
@@ -41,6 +41,7 @@
         "next-auth": "4.10.3",
         "next-plugin-preval": "1.2.4",
         "next-seo": "5.4.0",
+        "next-sitemap": "^3.1.21",
```

Comment:
We try to pin all of our non-dev dependencies, would be nice to have this follow suit.